test/NuGet.Core.Tests/NuGet.Commands.Test/SignCommandRunnerTests.cs

```diff
@@ -21,7 +21,7 @@ public SignCommandRunnerTests(CertificatesFixture fixture)
             _fixture = fixture;
         }
 
-        [Fact]
+        [PlatformFact(Platform.Windows, Platform.Linux)]
```

Comment:
Why not on Mac?

Reply:
There were some changes in how .NET cryptography works on macOS on netcore 2.1+ … therefore this tests started failing only on mac. As part of the XPLAT work we need to fix those issues and reenable tests for all OS. The task to track that is https://github.com/NuGet/Home/issues/7608 and https://github.com/NuGet/Home/issues/7606